css/src/components/form/form.scss

```
@@ -0,0 +1,238 @@
+/* stylelint-disable max-nesting-depth, selector-max-specificity  */
+
+$form-font-size: $font-size-7;
+$form-font-size-sm: $font-size-8;
+$form-font-size-lg: $font-size-6;
+
+$icon-size: 2.25em;
+
+$icon-color: $secondary-background !default;
+$icon-active-color: $secondary !default;
+
+.control {
+	position: relative;
+	font-size: $form-font-size;
+	text-align: left;
```

Could we use the logical value `start` here?

```suggestion
	text-align: start;
```

---

Updated